### ISSUE TITLE
Demo and Formula problem

### ISSUE THREAD

**Issue by SwimGeek** [unfoldadmin/django-unfold#131]
Hi

I was hoping to look at the demo site, but I can't find the username and password in your docs.
https://demo.unfoldadmin.com/admin/login/?next=/admin/

I then tried to install the formula demo, but I get an error when running 'poetry run python src/manage.py migrate':

'django.db.migrations.exceptions.MigrationSchemaMissing: Unable to create the django_migrations table (attempt to write a readonly database)'

It creates the sqlite db, but somehow it thinks that the db is read only.

python 3.11.5, django 4.2.5

**Comment by lukasvinclav** [comment]
Hey, I had to refactor piece of code to be sure that overridden admin site is really loaded. Updated the demo site and it should be hopefully fine. Both login and password are prefilled.

Regarding the issue with local migration, the problem was that I was forcing to have database in readonly mode. In production this is fine but on local, migrations were not passing. I added a condition to allow writing on local machines when DEBUG=1.

Let me know if you found something else :-)
